source/ftrack_connect_pipeline_qt/ui/assembler/base/__init__.py

```diff
@@ -667,26 +688,32 @@ def _definition_selected(self, index):
         self._set_default_mode()
 
     def _set_default_mode(self):
-        '''Find out from definition which is the default load mode and set it'''
+        '''Find out from which is the default load mode and set it'''
         mode = self._modes[0]
-        for stage in self.definition['components'][0]['stages']:
-            if stage['name'] == plugin._PLUGIN_IMPORTER_TYPE:
-                if not 'options' in stage['plugins'][0]:
-                    stage['plugins'][0]['options'] = {}
-                mode = stage['plugins'][0]['options'].get(
-                    'load_mode', 'import'
-                )
-                break
+        stage = self.definition.get_first(
+            type=core_constants.COMPONENT
+        ).get_first(
+            category=core_constants.STAGE,
+            name=core_constants.plugin._PLUGIN_IMPORTER_TYPE,
+        )
```

Comment:
Should't this work?
```
stage = self.definition.get_first(
            type=core_constants.COMPONENT
            category=core_constants.STAGE,
            name=core_constants.plugin._PLUGIN_IMPORTER_TYPE,
        )
```

Reply:
Yes, changing

Reply:
@henriknorin-ftrack Can't see the changes reflected.